src/nativefs_worker.js

```diff
@@ -0,0 +1,146 @@
+var files = [];
+var fd = 2;
+var sharedMemory;
+var fs_lock = -1;
+var nativefs_handle;
```

Comment:
I don't understand why this code has to be in worker.

Why can't the files be stored on the main thread?  It looks like its just an array of blobs which should work just as well on the main thread.

Reply:
Thank you very much for the review !
I thought it would be easier to have this version first. With Atomics.wait() impossible in the main thread, and having to do async reads instead of sync reads.

I agree it would be better to avoid having to send the file anywhere at all. I will send another version taking your comments into account.